bouncer/shared/utils.ts

```diff
@@ -548,6 +550,7 @@ export async function observeBroadcastSuccess(broadcastId: BroadcastId) {
       if (broadcastIdNumber === Number(event.data.broadcastId)) return true;
       return false;
     },
+    `observe Broadcast Success test tag: ${testTag}`,
```

Comment:
This print will be weird if `testTag` is undefined.

Reply:
fixed